service/gateway/response_util.go

```diff
@@ -30,6 +30,7 @@ var (
 	InvalidRange       = &errorDescription{errorCode: "InvalidRange", errorMessage: "Range is invalid", statusCode: http.StatusBadRequest}
 	InvalidAddress     = &errorDescription{errorCode: "InvalidAddress", errorMessage: "Address is illegal", statusCode: http.StatusBadRequest}
 	SignatureNotMatch  = &errorDescription{errorCode: "SignatureDoesNotMatch", errorMessage: "SignatureDoesNotMatch", statusCode: http.StatusForbidden}
+	SignatureExpired   = &errorDescription{errorCode: "SignatureExpired", errorMessage: "SignatureExpired", statusCode: http.StatusForbidden}
```

Comment:
`errorCode: "SignatureExpired"` to `errorCode: "AccessDenied"` maybe is better. errorCode has semantics, try to avoid too much customization, and try to keep it consistent with S3(refer to https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html#RESTErrorResponses). The more detailed internal reasons can be displayed through errorMessage.